src/components/Footer/Footer.jsx

```diff
@@ -0,0 +1,113 @@
+// React Imports
+import React from 'react';
+// Material UI Imports
+import Box from '@mui/material/Box';
+import Button from '@mui/material/Button';
+import Container from '@mui/material/Container';
+import Grid from '@mui/material/Grid';
+import Link from '@mui/material/Link';
+import Typography from '@mui/material/Typography';
+import { useTheme } from '@mui/material/styles';
+// Material Icons Imports
+import TwitterIcon from '@mui/icons-material/Twitter';
+import FacebookIcon from '@mui/icons-material/Facebook';
+import InstagramIcon from '@mui/icons-material/Instagram';
+
```

Comment:
Something minor, we don't have to do it in this PR in particular if we do want to add them. It's already explicit enough being named Footer, but I was thinking on if we would like to add some JSDoc to the file to be consistent. So something along the lines of:

```
/**
 * Footer Component - Footer Component for PASS
 * 
 * @memberof <something>
 * @name Footer
 */
```

Reply:
Oh yes. Sorry! First time working with JSdoc.  I will add, and also read into it more/make sure to add it to my development routine.

Reply:
Added